src/structures/ClientUser.js

```diff
@@ -43,6 +43,13 @@ class ClientUser extends User {
      * @type {Collection<string, string>}
      */
     this.notes = new Collection();
+
+    /**
+     * Discord client settings, such as guild positions
+     * <warn>This is only filled when using a user account.</warn>
+     * @type {Object}
```

Comment:
If this isn't optional on bot accounts it should be `{?Object}` to indicate it can be null